NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -267,17 +335,29 @@ public void CanSyncWithPoW()
 		}
 
 		[Fact]
-		public void CorrectCoinMaturity()
+		public async Task CorrectCoinMaturity()
 		{
 			using (var builder = NodeBuilderEx.Create())
 			{
 				var node = builder.CreateNode();
 				builder.StartAll();
 				node.Generate(builder.Network.Consensus.CoinbaseMaturity);
 				var rpc = node.CreateRPCClient();
-				Assert.Equal(Money.Zero, rpc.GetBalance());
-				node.Generate(1);
-				Assert.NotEqual(Money.Zero, rpc.GetBalance());
+				if (new[] {Liquid.Instance.Mainnet, Liquid.Instance.Testnet, Liquid.Instance.Regtest,}.Contains(
```

Comment:
The `GetBalanceAsync` should still pass and implies LBTC, even if it is liquid.

Reply:
GetBalance rpc call send a different json structure than bitcoind